dist/secure-handlebars.js

```diff
@@ -10658,15 +10697,100 @@ StrictContextParser.prototype.getCurrentState = function() {
 };
 
 
+// <iframe srcdoc=""> is a scriptable attribute too
+// Reference: https://html.spec.whatwg.org/multipage/embedded-content.html#attr-iframe-srcdoc
+var scriptableTags = {
+    script:1,style:1,
+    svg:1,xml:1,math:1,
+    applet:1,object:1,embed:1,link:1,
+    scriptlet:1                  // IE-specific
+};
```

Comment:
but that might be too restrictive, as people might do:
`<meta name="description" content="{{placeholder}}" >`
`<meta name="keywords" content="{{placeholder}}" >`